src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -106,37 +101,37 @@ void AdaptDispatch::_SetGraphicsOptionHelper(const DispatchTypes::GraphicsOption
         break;
     case DispatchTypes::GraphicsOptions::ForegroundBlue:
         s_DisableAllColors(attr, true); // turn off all color flags first.
-        attr |= FOREGROUND_BLUE;
+        WI_SetFlag(attr, FOREGROUND_BLUE);
         _changedForeground = true;
         break;
     case DispatchTypes::GraphicsOptions::ForegroundGreen:
         s_DisableAllColors(attr, true); // turn off all color flags first.
-        attr |= FOREGROUND_GREEN;
+        WI_SetFlag(attr, FOREGROUND_GREEN);
         _changedForeground = true;
         break;
     case DispatchTypes::GraphicsOptions::ForegroundCyan:
         s_DisableAllColors(attr, true); // turn off all color flags first.
-        attr |= FOREGROUND_BLUE | FOREGROUND_GREEN;
+        WI_SetFlag(attr, FOREGROUND_BLUE | FOREGROUND_GREEN);

```

Comment:
Probably needs SetAllFlags (and below anywhere there's >1)

Reply:
It seems to me that SetAllFlags should be used when flag is a variable. And SetFlag should be used when flag is compile time constant.

Reply:
I hear what you're saying, but WIL literally defines `WI_SetFlag` as `WI_SetAllFlags` with an assert about only setting one bit at a time. This is probably the reason why your builds are failing.

Reply:
I misunderstood it. Force pushed the fix.